pkg/sentry/fs/proc/task.go

```diff
@@ -293,21 +293,26 @@ func (e *exe) Readlink(ctx context.Context, inode *fs.Inode) (string, error) {
 // +stateify savable
 type namespaceSymlink struct {
 	ramfs.Symlink
+	inodeID uint64
```

Comment:
It doesn't seem this is used anywhere. Symlink target is calculated and stored in newNamespaceSymlink().